api/initializer-app.go

```diff
@@ -34,6 +34,10 @@ func StartServer() {
 		r.HandleFunc(route.Path, route.Handler).Methods(route.Method)
 	}
 
+	for _, route := range getAuthRoutes() {
+		r.HandleFunc(route.Path, route.Handler).Methods(route.Method)
+	}
+
```

Comment:
It's not related to this PR specifically, but you can make `GetDoctorRoutes` private by changing it to camelCase & keeping consistency with the other functions